Assignment2/rarycops-Miguel_Jorge.ttl

```diff
@@ -0,0 +1,18 @@
+@base <http://www.oeg-upm.net/>.
+@prefix c: <http://www.oeg-upm.net/class/>.
+@prefix s: <http://www.oeg-upm.net/sensor/>.
+@prefix co: <http://www.oeg-upm.net/computer/>.
+@prefix m: <http://www.oeg-upm.net/measurement/>.
+@prefix u: <http://www.oeg-upm.net/user/>.
+c:class:01 
```

Comment:
Not recommended to have two ":" Use only one before the prefix and substitute the another one with a hyphen or alike.

Reply:
solved